[BUG] ActionListTest.test_action_listincrements fails in sufficiently-negative-from-UTC timezones

## Bug summary

`ActionListTest.test_action_listincrements` checks a timestamp against a regex expecting a date in early January, 1970.  If the system timezone running the test is sufficiently negative from UTC, the test will start failing because the date shows up in late December, 1969 instead.  For instance (output trimmed for conciseness):

```
[build@arrakis rdiff-backup-2.2.5]$ timedatectl 
               Local time: Mon 2023-06-26 11:14:26 CDT
           Universal time: Mon 2023-06-26 16:14:26 UTC
                 RTC time: Mon 2023-06-26 16:14:26
                Time zone: US/Central (CDT, -0500)
System clock synchronized: yes
              NTP service: inactive
          RTC in local TZ: no
[build@arrakis rdiff-backup-2.2.5]$ python testing/action_list_test.py ActionListTest.test_action_listincrements --verbose --buffer 

<output trimmed>

======================================================================
FAIL: test_action_listincrements (__main__.ActionListTest.test_action_listincrements)
test the list increments action, without and with size
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/dv/build/arch/rdiff-backup/tmp/rdiff-backup-2.2.5/testing/action_list_test.py", line 131, in test_action_listincrements
    self.assertRegex(comtst.rdiff_backup_action(
AssertionError: Regex didn't match: b'---\n- base: increments.1970-01-0[12]T[0-9][0-9][:-][14]6[:-]40.*.dir\n  time: 10000\n  type: directory\n- base: bak\n  time: 20000\n  type: directory\n...\n\n' not found in b'---\n- base: increments.1969-12-31T20:46:40-06:00.dir\n  time: 10000\n  type: directory\n- base: bak\n  time: 20000\n  type: directory\n...\n\n'
```

Setting `TZ=utc` on this system, while running the test allows it to succeed:

```
[build@arrakis rdiff-backup-2.2.5]$ TZ=utc python testing/action_list_test.py ActionListTest.test_action_listincrements --verbose --buffer 
<trim>
----------------------------------------------------------------------
Ran 1 test in 0.923s

OK
```

## Version, Python, Operating System

Building from stock rdiff-backup-2.2.5 tar, on Python 3.11.3, on Arch Linux

## rdiff-backup call

n/a -- `python testing/action_list_test.py ActionListTest.test_action_listincrements --verbose --buffer`

## What happened and what did you expect?

Test run fails w/ negative-UTC timezone, but should succeed (or the test should fake the timezone itself)

## More information

Let me know if you need anything else!

--- a/testing/action_list_test.py
+++ b/testing/action_list_test.py
@@ -39,11 +39,11 @@ class ActionListTest(unittest.TestCase):
         # we backup twice to the same backup repository at different times
         comtst.rdiff_backup_action(
             True, True, self.from1_path, self.bak_path,
-            ("--api-version", "201", "--current-time", "10000"),
+            ("--api-version", "201", "--current-time", "111111"),
             b"backup", ())
         comtst.rdiff_backup_action(
             True, True, self.from2_path, self.bak_path,
-            ("--api-version", "201", "--current-time", "20000"),
+            ("--api-version", "201", "--current-time", "222222"),
             b"backup", ())
 
     def test_action_listfilesattime(self):
@@ -61,7 +61,7 @@ fileUnchanged
         self.assertEqual(comtst.rdiff_backup_action(
             True, None, self.bak_path, None,
             ("--api-version", "201"),
-            b"list", ("files", "--at", "10000"), return_stdout=True),
+            b"list", ("files", "--at", "111111"), return_stdout=True),
             b""".
 fileChanged
 fileOld
@@ -100,7 +100,7 @@ fileUnchanged
         self.assertEqual(comtst.rdiff_backup_action(
             True, None, self.bak_path, None,
             ("--api-version", "201"),
-            b"list", ("files", "--changed-since", "10000"), return_stdout=True),
+            b"list", ("files", "--changed-since", "111111"), return_stdout=True),
             b"""changed fileChanged
 new     fileNew
 deleted fileOld
@@ -133,11 +133,11 @@ deleted fileOld
             ("--api-version", "201", "--parsable"),
             b"list", ("increments", ), return_stdout=True),
             b"""---
-- base: increments.1970-01-0[12]T[0-9][0-9][:-][14]6[:-]40.*.dir
-  time: 10000
+- base: increments.1970-01-0[12]T[0-9][0-9][:-][25]1[:-]51.*.dir
+  time: 111111
   type: directory
 - base: bak
-  time: 20000
+  time: 222222
   type: directory
 ...
 
@@ -150,10 +150,10 @@ deleted fileOld
             b"list", ("increments", "--size"), return_stdout=True),
             b"""---
 - size: [0-9]+
-  time: 10000
+  time: 111111
   total_size: [0-9]+
 - size: [0-9]+
-  time: 20000
+  time: 222222
   total_size: [0-9]+
 ...
 

